source/quickstart/container.rst

```diff
@@ -6,35 +6,303 @@ Quickstart for Containers
+   
+            .. code-block:: shell
+               :class: copyable
 
-#. Step 2
+               podman run \
+                  -p 9000:9000 \
+                  -p 9001:9001 \
+                  -v ~/minio/data:/data \
+                  quay.io/minio/minio server /data --console-address ":9001"
+   
+            The example above works this way:
+   
+            - ``podman run`` starts the container.
+            - ``-p`` binds a local port to a container port.
+            - ``-v`` sets a file path as a persistent volume location for the container to use.'
+            - When MinIO writes to ``/data`` within the container, the data actually writes to the ``~/minio/data`` local file path.
+              You can set any file path to which the user has read, write, and delete permissions to use.
+   
+         .. tab-item:: Windows
+   
+            .. code-block:: shell
+               :class: copyable
+   
+               podman run \
+                  -p 9000:9000 \
+                  -p 9001:9001 \
+                  -v D:\data:/data \
+                  quay.io/minio/minio server /data --console-address ":9001"
+   
+            The example above works this way:
+   
+            - ``podman run`` starts the container.
+            - ``-p`` binds a local port to a container port.
+            - ``-v`` sets a file path as a persistent volume location for the container to use.'
+            - When MinIO writes to ``/data`` within the container, the data actually writes to the ``D:\data`` local file path.
+              You can set any file path to which the user has read, write, and delete permissions to use.
+   
+   .. dropdown:: Docker as ``root``
+      :name: docker-rootfull
+   
+      .. tab-set::
+   
+         .. tab-item:: GNU/Linux or MacOS
+   
+            .. code-block:: shell
+               :class: copyable
+   
+               mkdir -p ~/minio/data
+   
+               docker run \
+                  -p 9000:9000 \
+                  -p 9001:9001 \
+                  --name minio \
+                  -v ~/minio/data:/data \
+                  -e "MINIO_ROOT_USER=AKIAIOSFODNN7EXAMPLE" \
+                  -e "MINIO_ROOT_PASSWORD=wJalrXUtnFEMI/K7MDENG/bPxRfiCYEXAMPLEKEY" \
+                  quay.io/minio/minio server /data --console-address ":9001"
+         
+            The example above works this way:
+   
+            - The ``mkdir`` command creates a new local directory at ``~/minio/data`` in your home directory.
+            - The ``docker run`` command starts the MinIO container with the ``-v`` argument to map the local path (``~/minio/data``) to the specified virtual container directory (``/data``).
+            - When MinIO writes data to ``/data``, that data actually writes to the local path ``~/minio/data`` where it can persist between container restarts.
+              You can replace ``~/minio/data`` with another local file location to which the user has read, write, and delete access.
+                 
```

Comment:
If we keep the `-e` lines we should let users know this is setting envvars corresponding to :envvar:`MINIO_ROOT_USER` and :envvar:`MINIO_ROOT_PASSWORD`